### PR TITLE
Improved documentation for the exercises

### PR DESCRIPTION
**Pull request by sponce**
- moved the content of the tex file presenting exercises into the README.md
- dropped the CheatSheet for the essential course
- created Schedule files for the essential and advanced course listing exercises to be done there, per day

### CODE REVIEW

File: code/README.md [hunk 1]
```diff
@@ -1 +1,101 @@
...
+
+How to test your setup
+----------------------
+
+### Just use hello exercise
+
+ - go to code/hello
+ - follow the README
+
+### Practically for compilation
```
Comment (hageboeck) [inline code/README.md:72]:
```suggestion
### How to compile?
```

File: code/README.md [hunk 2]
```diff
@@ -1 +1,101 @@
...
+
+  - cd code/hello
+  - make;
```
Comment (hageboeck) [inline code/README.md:75]:
```suggestion
  - make
```

File: code/README.md [hunk 3]
```diff
@@ -1 +1,101 @@
...
+### Practically for valgrind/cppcheck
+
+  - valgrind --tool=callgrind ./hello; kcachegrind
+  - cppcheck .
+
+Using lxplus
+------------
+
+  - lxplus is perfectly suitable for the exercises
+  - is has all the needed compilers/tools in cvmfs
```
Comment (hageboeck) [inline code/README.md:88]:
```suggestion
  - it has all the needed compilers/tools in cvmfs
```

File: code/README.md [hunk 4]
```diff
@@ -1 +1,101 @@
...
+
+### Practically on lxplus8
```
Comment (bernhardmgruber) [inline code/README.md:90]: So, would you merge the Windows guide into this README as well?

Comment (sponce) [reply code/README.md:90]: Once this is merged, I would say yes. At this stage, I'm fighting md for the schedule files. And gitlab md is special and cannot relaly be debugged locally... a pleasure :-)

**Comment by sponce** [comment]
ok, this should be ready to merge. So that we have it for this afternoon

**Comment by sponce** [comment]
Jobs are stuck in github right now, so merging this as we need it for the meeting at 2pm. We'll fix in a later MR the eventual problems

**Pull request merged**